data/transactions/asset_test.go

```diff
@@ -0,0 +1,182 @@
+package transactions
```

Comment:
This file needs the standard license boilerplate.

Reply:
I've added what's in `scripts/LICENSE_HEADER` to the top of the file so hopefully the check doesn't fail. But it looks like none of the other files in the same folder have it either. In fact, very few `.sh` files do:
```sh
% find . -type f -iname "*.sh" | xargs grep -li "copyright"
./cmd/tealdbg/bundle_home_html.sh
./crypto/copies/darwin/arm64/libsodium-fork/build-aux/ltmain.sh
./test/scripts/e2e_subs/asset-misc.sh
./scripts/build_deb.sh
./scripts/release/mule/package/deb/package.sh
./scripts/release/build/deb/build_deb.sh
./scripts/check_license.sh
./scripts/travis/travis_wait.sh
```